lib/ansible/module_utils/redfish_utils.py

```diff
@@ -0,0 +1,944 @@
+# Copyright (c) 2017-2018 Dell EMC Inc.
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import os
+import json
+import re
+import xml.etree.ElementTree as ET
+from distutils.version import LooseVersion
+from datetime import datetime
```

Comment:
lines 10-12 are imported by not used (unless I'm missing something)

Reply:
Good catch. Probably used it at some point and then changed code. Will remove.